web_tree_dynamic_colored_field/README.rst

```diff
@@ -81,26 +81,7 @@ Usage
       </field>
       ...
 
-      With this example, column which renders 'name' field will have its text colored in white on a customer records.
-
--  In the tree view declaration, use
-   ``options='"color_field": "my_color"'`` attribute in the ``tree``
-   tag:
-
-   ::
-
-      ...
-      <field name="arch" type="xml">
-          <tree string="View name" colors="color_field: my_color" >
-              ...
-              <field name="my_color" invisible="1"/>
-              ...
-          </tree>
-      </field>
-      ...
-
--  You can also use ``colors="bg_color_field: my_color"`` to defined the
-   field name that will be used for the background color of the line.
```

Comment:
Why this example is removed?

Reply:
I thought it refers to the deprecated functionality... (Oct 14, 2022)
https://github.com/OCA/web/blob/15.0/web_tree_dynamic_colored_field/readme/ROADMAP.rst
https://github.com/OCA/web/commit/4b1f3fe09c1bd7d9d7220b9b2d8b8ba2c7e09a69

Now I noticed the removal was kind of "reverted" 3 days later (Oct 17, 2022)
https://github.com/OCA/web/commit/9994bc636b11bd1764935763e518c66c26d7cb4b

So: it is confusing, we didn't use it in Siramis, and I didn't want to spend time on this...
May be you have ideas how to handle this?

PS: whole JS code history: https://github.com/OCA/web/commits/15.0/web_tree_dynamic_colored_field/static/src/js/web_tree_dynamic_colored_field.js

Reply:
Not sure why you need to touch this..:)

Reply:
Well usually when we migrate OCA modules, we try to cover whole functionality, not only the one we need. Your PR might be rejected (not so bad, since we have our own branch).
So let's discuss. @oerp-odoo @DarkoNikolovski should we invest more time to migrate whole functionality?

Reply:
If we can spare some time to properly migrate a module, then we should do it. Maybe @DarkoNikolovski can comment more on that:)

Reply:
@dz0 Is it possible to migrate the missing part? If yes what would be the estimated time for that?

Reply:
seems the functionality can't work, because of reason in [ROADMAP](https://github.com/OCA/web/blob/15.0/web_tree_dynamic_colored_field/readme/ROADMAP.rst).

tried on 17, the [schema](https://github.com/odoo/odoo/blob/17.0/odoo/import_xml.rng)  does  not allow  `colors` attr for `tree`
![paveikslas](https://github.com/versada/web/assets/1269555/90b0a6b6-207e-491c-945c-507e6fa58a4d)

Reply:
If not possible then let's continue as it is.
Wondering how it is working in v.15  or is it working at all, because I can't see `colors` there as well